runtime/src/lib.rs

```diff
@@ -285,6 +285,7 @@ impl pallet_creditcoin::Config for Runtime {
 	type InternalPublic = sp_core::sr25519::Public;
 	type HashIntoNonce = Hash;
 	type UnverifiedPoolsLimit = ConstU32<10000>;
+	type UnverifiedTransferTimeout = ConstU32<60>;
```

Comment:
I think when adding new types we should also remove the typedefs under `creditcoin-js`.